### PR TITLE
Added global is_ironpython variable

### PR DESCRIPTION
**Pull request by maxcapodi78**
Corrected a bug in Edb when no input aedb is provided

### CODE REVIEW

File: pyaedt/__init__.py [hunk 1]
```diff
@@ -1,6 +1,18 @@
 import os
 # Import exception handling here due to:
 # https://github.com/pyansys/PyAEDT/pull/243
+import sys
+
+is_ironpython = False
+if "IronPython" in sys.version or ".NETFramework" in sys.version:
+    is_ironpython = True
```
Comment (akaszynski) [inline pyaedt/__init__.py:8]: This is a bit cleaner:

```suggestion
is_ironpython = "IronPython" in sys.version or ".NETFramework" in sys.version
```

File: pyaedt/siwave.py [hunk 2]
```diff
@@ -11,10 +11,9 @@
 import pkgutil
 
 from .misc import list_installed_ansysem
+from pyaedt import is_ironpython,_pythonver
```
Comment (akaszynski) [inline pyaedt/siwave.py:14]:
```suggestion
from pyaedt import is_ironpython, _pythonver
```

File: pyaedt/application/DataHandlers.py [hunk 3]
```diff
@@ -215,3 +219,307 @@ def random_string(length=6, only_digits=False, char_set=None):
             char_set = string.ascii_uppercase + string.digits
     random_str = ''.join(random.choice(char_set) for _ in range(int(length)))
     return random_str
+
+
+def unique_string_list(element_list, only_string=True):
+    """
```
Comment (akaszynski) [inline pyaedt/application/DataHandlers.py:225]:
```suggestion
    """Return a list of strings.
```

**Review by Alberto-DM — APPROVED** [review]
Looks good to me

**Pull request merged**